# Reproduce the Bug

# # Reproduce the Bug
# from random import randint
# dice_imgs = ["❶", "❷", "❸", "❹", "❺", "❻"]
# dice_num = randint(1, 6) - > this will give an error due to list = 0-5 not 1-6
# print(dice_imgs[dice_num])


from random import randint

dice_imgs = ["❶", "❷", "❸", "❹", "❺", "❻"]

dice_num = randint(0, 5)

print(dice_imgs[dice_num])
